src/models/stores/supports.ts

```diff
@@ -38,6 +40,27 @@ export const TeacherSupportModel = types
         self.visible = visible;
       }
     };
+  })
+  .views((self) => {
+    return {
+      get sectionTarget(): TeacherSupportSectionTarget {
+        return self.type === SupportItemType.section
```

Comment:
Reminder that this function is useful because the MST model stores section target a little differently to match up with default supports. That is, it knows if it is a `section` or a `problem` level support rather than having `all` or a sectionId as its target.